compute/cloud-client/src/main/java/compute/disks/storagepool/CreateHyperdiskStoragePool.java

```diff
@@ -48,16 +48,17 @@ public static void main(String[] args)
     long provisionedIops = 3000;
     // the throughput in MBps to provision for the storage pool.
     long provisionedThroughput = 140;
+    String performanceProvisioningType = "advanced";
```

Comment:
if this value is constant, please use it in the code sample method directly. if this value is one of the fixed list of values, please add comment referencing documentation that shows the list of possible values.

Reply:
Added comments. Thank you!